src/aioquic/quic/connection.py

```diff
@@ -233,6 +234,7 @@ def __init__(
         self._ack_delay = K_GRANULARITY
         self._close_at: Optional[float] = None
         self._close_event: Optional[events.ConnectionTerminated] = None
+        self._reset_event: Optional[events.StreamReset] = None
```

Comment:
This doesn't seem to be used? Also, let's try and keep things sorted alphabetically :)